system/CLI/CLI.php

```diff
@@ -457,7 +457,7 @@ public static function print(string $text = '', ?string $foreground = null, ?str
      */
     public static function write(string $text = '', ?string $foreground = null, ?string $background = null)
     {
-        if ($foreground || $background) {
+        if ($foreground !== null || $background !== null) {
```

Comment:
same here

Reply:
Maybe you should use a cast? There are many such updates, especially for `empty() `- returns `null, ", '0', 0, false`

Reply:
Yes, cast to string is fine. For the others, let's check if the casted value is acceptable.

Reply:
I wouldn't use `null `unless it's expecting an object. In most cases, scalar types of `string int` are specified...